x-pack/solutions/search/plugins/search_indices/common/routes.ts

```diff
@@ -13,3 +13,5 @@ export const POST_CREATE_INDEX_ROUTE = '/internal/search_indices/indices/create'
 export const INDEX_DOCUMENT_ROUTE = '/internal/search_indices/{indexName}/documents/{id}';
 
 export const SEARCH_DOCUMENTS_ROUTE = '/internal/search_indices/{indexName}/documents/search';
+
+export const GET_ONBOARDING_DATA_ROUTE = '/internal/search_indices/onboarding_data';
```

Comment:
Similar to query key question. Currently there is no usage for this route but wondering was this added for future usage?